apkid/rules/elf/obfuscators.yara

```diff
@@ -215,3 +215,19 @@ rule avdobfuscator : obfuscator
     any of them and is_elf
 }
 
+rule metafortress : obfuscator
+{
+  meta:
+    description = "MetaFortress"
+    url         = "https://www.insidesecure.com/Products/Application-Protection/Software-Protection/Code-Protection"
+    sample      = "326632f52eba45609f825ab6746037f2f2b47bfe66fd1aeebd835c8031f4fdb0"
+    author      = "Eduardo Novella"
+
+  strings:
+    $a = { 00 4d65 7461 466f 7274 7265 7373 3a20 2573 0025 733a 2025 730a 00 } // MetaFortress %s.%s: %s
+    $b = { 00 4d65 7461 466f 7274 7265 7373 00 } // MetaFortress
+    $c = "METAFORIC"
```

Comment:
I'm seeing a general trend on over reliance on strings, especially naked strings. The main problem with these rules isn't that they may FP -- though it's possible -- it's that _other_, much better rules are possible. I don't think it's worth it to add _relatively_ weak rules when a little extra effort could make something much better.

Reply:
Yup agree. Let me find new samples to tweak a bit better the rule.

Reply:
Any new samples used to revise this? Or is it all set for next review?

Reply:
yeah! ETA soon